superset/config.py

```diff
@@ -986,6 +989,14 @@ def CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC(  # pylint: disable=invalid-name
 # See here: https://github.com/dropbox/PyHive/blob/8eb0aeab8ca300f3024655419b93dad926c1a351/pyhive/presto.py#L93  # pylint: disable=line-too-long,useless-suppression
 PRESTO_POLL_INTERVAL = int(timedelta(seconds=1).total_seconds())
 
+# Allow list of custom authentications for each DB engine.
+ALLOWED_EXTRA_AUTHENTICATIONS: Dict[str, List[str]] = {
+    "trino": [
+        # "your.module:AuthClass",
+        # "another.extra:auth_method",
+    ],
```

Comment:
let's leave the default empty - I'd prefer to have it as a commented example above. Something like
```python
# Allow list of custom authentications for each DB engine.
# Example:
# ALLOWED_EXTRA_AUTHENTICATIONS: Dict[str, List[str]] = {
#     "trino": [
#         "your.module:AuthClass",
#         "another.extra:auth_method",
#     ],
# }
ALLOWED_EXTRA_AUTHENTICATIONS: Dict[str, List[str]] = {}
```